alerting/src/main/kotlin/org/opensearch/alerting/aggregation/bucketselectorext/BucketSelectorExtAggregator.kt

```diff
@@ -143,7 +143,7 @@ class BucketSelectorExtAggregator : SiblingPipelineAggregator {
     private fun isAccepted(obj: Any, filter: IncludeExclude?): Boolean {
         return when (obj.javaClass) {
             String::class.java -> {
-                val stringFilter = filter!!.convertToStringFilter(DocValueFormat.RAW)
+                val stringFilter = filter!!.convertToStringFilter(DocValueFormat.RAW, null)
```

Comment:
Just to clarify, wasn't this change made in another PR?

Reply:
yeah will rebase this